.travis.yml

```diff
@@ -9,16 +9,19 @@ env:
 stages:
   - test
   - lint
-  
+
 
 matrix:
   include:
     - python: 2.7
       env: TOXENV=2.7
+      before_install: sudo apt-get update && sudo apt-get install libgnutls-dev
```

Comment:
Why do we need to re-add this?

Reply:
Travis build was failing without this package. See: https://travis-ci.org/celery/kombu/jobs/568362312